include/unifex/stop_immediately.hpp

```diff
@@ -86,7 +86,7 @@ struct stop_immediately_stream {
           stream_.stopSource_.request_stop();
           auto receiver = std::exchange(stream_.nextReceiver_, nullptr);
           assert(receiver != nullptr);
-          std::move(*receiver).done();
+          std::move(*receiver).set_done();
```

Comment:
could use the cpo's for calls to `std::move(*receiver)`, yes?

Reply:
This particular receiver is a pointer to a type-erased base-class `next_receiver_base` for which we know the interface.

Also, this type, while it has the same interface as a receiver is never actually used as a receiver (it's never passed to `connect()`).

So I didn't think it necessary to use the CPOs here.